google/ggrc-core
# GGRC-5926 States are not preselected in Global Search if open it on My Assessments page

Add preselected states in Global Search for 'My Assessments' page



# Issue description

States are not preselected in Global Search if open it on My Assessments page

# Steps to test the changes

1. Log in ti GGRC and open My Assessments page 
2. Click on Global Search button 
3. Look at the state field 

Actual Result: States are not preselected in Global Search if open it on My Assessments page 
Expected Result: States should be preselected in Global Search if open it on My Assessments page

# Solution description

Change condition for preselected states on "My Assessments" page in "Global Search" modal   

# Sanity checklist




# PR Review checklist
